src/java/com/blackNecoProject/pages/NewCustomer.java

```diff
@@ -0,0 +1,49 @@
+package com.blackNecoProject.pages;
+
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebDriver;
+
+public class NewCustomer extends BasePage {
+        private By NewCustomer =By.linkText("New Customer");
+        private By FormCustomer=By.name("addcust");
+        private By Name =By.name("name");
+        private By Gender =By.name("rad1");
+        private By Birth =By.name("dob");
+        private By Address=By.name("addr");
+        private By City =By.name("city");
+        private By State =By.name("state");
+        private By Pin =By.name("pinno");
+        private By Telephone=By.name("telephoneno");
+        private By EmailCustomer=By.name("emailid");
+        private By PasswordCustomer=By.name("password");
+        private By SubmitCustomer=By.name("sub");
+        private By FormRegist=By.name("regmsg");
+        public NewCustomer(WebDriver driver){
+        super(driver);
+    }
+
+    public void enterNewCustomer(){
+            click(NewCustomer);
+    }
+    public By FormCustomer() {
```

Comment:
What do this method???

Reply:
it is necessary to check presence of the form new customer